modules/raft/src/main/java/org/apache/ignite/raft/jraft/storage/logit/option/StoreOptions.java

```diff
@@ -17,23 +17,27 @@
 
 package org.apache.ignite.raft.jraft.storage.logit.option;
 
+import org.apache.ignite.internal.lang.IgniteSystemProperties;
+import org.apache.ignite.internal.util.Constants;
 import org.apache.ignite.raft.jraft.storage.logit.storage.file.FileHeader;
 import org.apache.ignite.raft.jraft.storage.logit.storage.file.index.IndexFile.IndexEntry;
 
 /**
  * Storage options
  */
 public class StoreOptions {
+    public static final String LOGIT_STORAGE_SEGMENT_FILE_SIZE_PROPERTY = "LOGIT_STORAGE_SEGMENT_FILE_SIZE";
```

Comment:
What do you think, maybe we should do something similar to Ignite 2 and add the prefix "IGNITE_" to such properties?

Reply:
Yes, I think it's a good idea